templates/vue-demo-store/components/checkout/CheckoutCartItem.vue

```diff
@@ -1,51 +1,51 @@
 <script setup lang="ts">
-import { getSmallestThumbnailUrl } from "@shopware-pwa/helpers-next";
-import { LineItem } from "@shopware-pwa/types";
+  import { getSmallestThumbnailUrl } from "@shopware-pwa/helpers-next";
+  import { LineItem } from "@shopware-pwa/types";
```

Comment:
script setup components have formatting from the first line without tabs, can you make sure you formatted this file? :)

Reply:
idk why but I had a prettier as default formatter. Switched back to volar and works good as expected :)